examples/chime/voiceconnector.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: chime.aws.upbound.io/v1beta1
+kind: VoiceConnector
+metadata:
+  annotations:
+    meta.upbound.io/example-id: chime/v1beta1/voiceconnector
+  labels:
+    testing.upbound.io/example-name: test
+  name: test
+spec:
+  forProvider:
+    awsRegion: us-west-1
```

Comment:
Looks like AWS does not allow `awsRegion` to be set to `us-west-1`:
```
1.66980215229754e+09    DEBUG   provider-aws    Cannot observe external resource        {"controller": "managed/chime.aws.upbound.io/v1beta1, kind=voiceconnector", "request": "/test", "uid": "9242f53c-03aa-4851-bb99-c9b8a3d89e17", "version": "2699", "external-name": "test", "error": "cannot run refresh: refresh failed: expected aws_region to be one of [us-east-1 us-west-2], got us-west-1: ", "errorVerbose": "refresh failed: expected aws_region to be one of [us-east-1 us-west-2], got us-west-1: \ncannot run refresh
```

According to Terraform docs, this field is optional. Could you please remove it from the example manifests?
```suggestion
```